### ISSUE TITLE
Fix the NaN training loss in Llama 3

### ISSUE THREAD

**Issue by tengyifei** [AI-Hypercomputer/torchprime#9]
Note that Llama 2 runs fine
